assets/app/views/modals/debug-terminal.html

```diff
@@ -34,7 +34,15 @@ <h2 ng-if="debugPod.status.phase !== 'Failed'" class="text-muted">
       </div>
       <div ng-if="container | entrypoint : image" class="original-cmd-msg">
         <label>Original Command:</label>
-        <code>{{container | entrypoint : image}}</code>
+        <code>
+          <truncate-long-text
+              content="container | entrypoint : image"
+              limit="80"
+              newline-limit="1"
+              expandable="false"
```

Comment:
Is there enough room in a dialog for a really long command? You can hover your mouse and see it for what it's worth.